packages/@ourworldindata/grapher/src/controls/Checkbox.tsx

```diff
@@ -4,24 +4,26 @@ import { faCheck } from "@fortawesome/free-solid-svg-icons"
 
 export class Checkbox extends React.Component<{
     checked: boolean
-    label: string
     onChange: () => any
+    label: string | (() => React.ReactNode)
```

Comment:
out of interest, why is the label a function and not a plain react node?

Reply:
thanks @sophiamersmann!

Yes you're right, we don't need the render prop here 👍